journal.py

```diff
@@ -110,15 +112,18 @@ def write_entry(request):
     request.db.cursor().execute(INSERT_ENTRY, [title, text, created])
 
 
-@view_config(route_name='add', request_method='POST')
+@view_config(route_name='add', renderer='json')
```

Comment:
Any reason why you removed the predicate requiring POST for this view?  You aren't processing anything if the request comes in as GET.  In fact, it looks to me like nothing will return on a GET request, which will raise a 500 error anyway, so you should probably just restricte the view to POST only.